bot/bot.go

```diff
@@ -48,6 +48,8 @@ func run(bot *tgbotapi.BotAPI) {
 
 			if strings.HasPrefix(callbackText, "lectures_") {
 				handleCallback(bot, &update, "lezioni", callbackText)
+			} else if strings.HasPrefix(callbackText, "representatives_") {
+				handleCallback(bot, &update, "rappresentanti", callbackText)
```

Comment:
forse ci sta fare una funzione che dato il prefisso chiama il comando handleCallback con il prefisso.

Reply:
@foxyseta dato che a te piace astrarre magari sei d'accordo. Altrimenti lasciamo così.

Reply:
Possiamo estrarre il prefisso fino al `_` e chiamare `handleCallback` con quel prefisso. Tanto se non viene trovato non succede nulla. Se però non esiste un prefisso nel `callbackText` non chiamiamo nulla.

Reply:
Scusa ma non mi è chiaro il messaggi. Il comando è uno soltanto uno: `/rappresentanti`. Da li vengono mostrati i bottoni in modo da poter scegliere il corso di laurea di cui si vogliono i rappresentanti.
![2023-12-05T15:30:21,909007349+01:00](https://github.com/csunibo/informabot/assets/115153426/c66bb35a-30d2-4477-ad17-b19209f2153a)

Reply:
ah lol come non detto scusa. Avevo frainteso male il contesto e non ancora cominciato a fare la recensione. Ci penso ora.